psi4/driver/driver_cbs.py

```diff
@@ -25,34 +25,152 @@
 #
 # @END LICENSE
 #
+"""Plan, run, and assemble QC tasks to obtain composite method, basis, & options treatments.
+
+===
+CBS
```

Comment:
Repeat my request for additional user guidance, e.g., "CBS Flow", that I made on FINDIF.